### DISCUSSION TITLE
Problem with hardware switch

### DISCUSSION THREAD

**Discussion by cr4shydlo** [Blackymas/NSPanel_HA_Blueprint#834] [Q&A]
Hello. I have first switch as wirtual (run WLED) and sec. one as phisical lamp. 
When i try to run this sec. i need to wait 1-2sec before its run... at start all was ok. I think this show up after update to 3.3

What i can check to repair it ?

**Comment by edwardtfn** [comment]
I will need your help to troubleshoot this...

1. As I understood, the problem is when you press the right physical button it is taking 1-2sec for the physical relay to react, is that correct?
2. If you go to the device page related to your panel in Home Assistant (Settings > Devices and Services > ESPHome > your panel), you will see a sensor related to the right button. When you press that button on the panel, do you see the same delay for this sensor to change state in Home Assistant?
3. On that same page, you have the two switches related to the physical relays. Doy you see any lag from changing one of those relays and your light turning on/off?

**Comment by cr4shydlo** [comment]
Hello.
1. Correct.
2. Yes, here is lag too.
3. Here too.

But now i power restart nspanel, and for now all works ok. I dont know why. I wait couple days, and let know in this topic. 
Best regards, and thanks for reply.

**Reply by edwardtfn** [reply]
Sounds like some issue with the network but that shouldn't improve with a power restart...
